src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java

```diff
@@ -600,4 +602,10 @@ public CaseData setIssueDate() {
      *  Withdraw request flag.
      */
     private String isWithdrawRequestSent;
+
+    /**
+     *  Service Of Application.
+     */
+    private OrdersToServeSA serviceOfApplicationScreen1;
+    private ConfirmRecipients confirmRecipients;
```

Comment:
Is it possible to move them under a single object in order to avoid the conflict with AllArgsConstructor